syntax = "proto3";

message CommandRequest {
  int32 code = 1;
}
